media/accessible.css

```diff
@@ -0,0 +1,4 @@
+.blocklyTable {
+  vertical-align: top;
+}
+.blocklyTree .blocklyActiveDescendant > label, .blocklyActiveDescendant > button, .blocklyActiveDescendant > input {outline: 2px dotted #00f;}
```

Comment:
Separate selectors and declarations by new lines.
Always start a new line for each selector and declaration.
https://google.github.io/styleguide/htmlcssguide.xml?showone=Selector_and_Declaration_Separation#Selector_and_Declaration_Separation